### PR TITLE
[AptosFramework] unified with core framework

### PR DESCRIPTION
**Pull request by zekun000**
Remove the marker in the core framework to get rid of all the boilerplate code.

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/Account.move [hunk 1]
```diff
@@ -66,7 +85,6 @@ module AptosFramework::Account {
         currency_code_required: bool,
     ) {
         assert!(Signer::address_of(account) == @CoreResources, Errors::requires_address(ENOT_APTOS_FRAMEWORK));
-        move_to(account, Marker<T> {});
```
Comment (areshand) [inline aptos-move/framework/aptos-framework/sources/Account.move:69]: just a question. what was the original intention for introducing marker?

Comment (areshand) [inline aptos-move/framework/aptos-framework/sources/Account.move:69]: just a question. what was the original intention for introducing marker?

**Comment by areshand** [comment]
lgtm. please resolve the conflicts.

**Comment by areshand** [comment]
lgtm. please resolve the conflicts.

File: aptos-move/framework/aptos-framework/sources/Account.move (hunk 1 above)
Comment (zekun000) [reply aptos-move/framework/aptos-framework/sources/Account.move:69]: to support multi framework I think

**Review by areshand — APPROVED** [review]

**Pull request closed without merging**